fastchat/utils.py

```diff
@@ -65,11 +65,11 @@ def build_logger(logger_name, logger_filename):
     )
     handler.setFormatter(formatter)
 
-    for logger in [stdout_logger, stderr_logger, logger]:
-        if logger in visited_loggers:
+    for i in [stdout_logger, stderr_logger, logger]:
```

Comment:
use another name. Such as `l`?

Reply:
I will accept your suggestions.
I am not a professional programmer, and my habit is to use `i` to represent temporary variables. I don’t know any other better names either.